src/main/java/org/subethamail/smtp/client/SMTPClient.java

```diff
@@ -321,7 +321,7 @@ public SocketAddress getLocalSocketAddress() {
      * @return a nice pretty description of who we are connected to
      */
     public String getHostPort() {
-        return this.hostPortName.get();
+        return this.hostPortName.orElse(null);
```

Comment:
This was needed for tests. Incorrectly it can be invoked with an empty option generating exeptions.